src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMapToInt(x -> Arrays.stream(x.split(","))
+                        .mapToInt(Integer::valueOf))
+                .filter(x -> x % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException(
+                        "Can't get min value from list: "
+                                + numbers
+                ));
```

Comment:
Minor formating mishap

```suggestion
        return numbers.stream()
                .flatMapToInt(x -> Arrays.stream(x.split(","))
                .mapToInt(Integer::valueOf))
                .filter(x -> x % 2 == 0)
                .min()
                .orElseThrow(() -> new RuntimeException(
                        "Can't get min value from list: "
                                + numbers
                ));
```